src/pkcs1v15/signature.rs

```diff
@@ -1,3 +1,4 @@
+#[allow(unused_imports)]
```

Comment:
Wondering why we need this?

Reply:
My hypothesis was that upstream was only warning on unused_imports whereas the risc0 repo has it set to error. But I'm not very familiar with how Rust warning/error settings work so I'm not sure if this sort of cross-crate setting contamination can even happen.